### ISSUE TITLE
Calendar Not Loading after upgrading to WP 5.5

### ISSUE THREAD

**Issue by pikurasa** [bookingactivities/booking-activities#76]
Calendar not visible (tested on Calendar Editor and Bookings Pages)

I see this in the logs:

```
This page uses the non standard property “zoom”. Consider using calc() in the relevant property values, or using “transform” along with “transform-origin: 0 0”. admin.php
Uncaught TypeError: jQuery(...).live is not a function
    <anonymous> https://online.mapflc.com/wp-content/plugins/sprout-invoices-pro/resources/admin/js/sprout_invoice.js?ver=19.9.1.1:31
    jQuery 4
sprout_invoice.js:31:44
MouseEvent.mozPressure is deprecated. Use PointerEvent.pressure instead. tinymce.min.js:2:8856
Uncaught TypeError: s is undefined
    jQuery 13
load-scripts.php:349:479
Uncaught TypeError: s is undefined
    jQuery 13
load-scripts.php:349:479
Uncaught TypeError: s is undefined
    jQuery 13
load-scripts.php:349:479
Uncaught TypeError: s is undefined
    jQuery 13
load-scripts.php:349:479
Uncaught TypeError: s is undefined
    jQuery 13
load-scripts.php:349:479
Uncaught TypeError: s is undefined
    jQuery 13
load-scripts.php:349:479
Uncaught TypeError: s is undefined
    jQuery 13
load-scripts.php:349:479
```

Sprout Invoices is another plugin I use (and one of the plugins identified at the top of the log). If you want, I could disable to see if I get the same issue with that plugin disabled.

**Comment by pikurasa** [comment]
Oddly enough the customer-facing calendars seem to be working.

https://online.mapflc.com/product/music-code-hackathon/

Perhaps this is just a conflict???

**Comment by pikurasa** [comment]
I don't know exactly what I did to fix it (although I did update some other plugins), but it is now fixed.

**Comment by yoancutillas** [comment]
Wordpress 5.5 droped jquery-migrate,
the problem was due to the "sprout-invoices-pro" plugin, they fixed it in 19.9.2 ([changelog](https://fr.wordpress.org/plugins/sprout-invoices/#developers))

Since javascript is single threaded, if there is a fatal error somewhere, everything stops working.
